appveyor.yml

```diff
@@ -0,0 +1,28 @@
+version: 1.0.{build}
+image: Visual Studio 2017
+pull_requests:
+    do_not_increment_build_number: true
+branches:
+    only:
+        - master
+configuration: Release
+build_script:
+    - cmd: dotnet build -c Release src/HtmlAgilityPack.vs2019.sln
+#test_script:
+#    - cmd: dotnet test -c Release
+#after_test:
+#    - choco install opencover.portable
+#    - choco install codecov
+#    - OpenCover.Console.exe -register:user -target:"C:\Program Files\dotnet\dotnet.exe" -targetargs:"test -c Release --framework net472 --verbosity q"
+#    - codecov -f "results.xml"
+artifacts:
+    - path: '**\*.nupkg'
+      name: nupkg
+    - path: '**\*.snupkg'
+      name: snupkg
```

Comment:
About snupkg: https://blog.nuget.org/20181116/Improved-debugging-experience-with-the-NuGet-org-symbol-server-and-snupkg.html